### DISCUSSION TITLE
[feed] Emoji Reactions

### DISCUSSION THREAD

**Discussion by sirinath** [fomantic/Fomantic-UI#1452] [Views (card, comment, item, statistic,...)] [answered]
Is it possible to have Emoji reactions on feeds?

**Comment by lubber-de** [comment, marked as answer]
Sure, just add them how you want them , for example in the meta group.
Here is something you can play with
https://jsfiddle.net/t07qhdLx/1/

![emojifeed](https://user-images.githubusercontent.com/18379884/81068285-67972a80-8ee0-11ea-9056-2af2854ef1c8.gif)
